### PR TITLE
Add openstack security webinar landing page

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add openstack security webinar landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/openstack-security-webinar](http://0.0.0.0:8001/engage/openstack-security-webinar)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Make sure it matches the [brief](https://docs.google.com/document/d/1Qg3sgDrvH9w_sgJpb2fp5lyOA5R-xhGZM3aKcyuQT68/edit?ts=5cbee163#heading=h.1v2sl7iz5yrb)

## Issue / Card

Fixes [#1100](https://github.com/canonical-web-and-design/web-squad/issues/1100)

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ] `h1` is missing 'Private cloud security'
- [ ] 'How Charmed OpenStack is decreasing common security concerns' should be `h4`
- [ ] delete button as we don't need or have one for this banner on the brief
- [ ] the hero is too deep (padding 96px). can we match with the 64px? check our [templates](https://www.ubuntu.com/engage/_heading-help)
- [ ] The BrightTalk iFrame is missing but I asked @caldav help. We should get back to us later today

**Comment by anasereijo** [comment]
@solazio looking good! 
Just 1 more small thing:
I like the ticked` p-list__item is-ticked` but the 2nd level (last 2 items) shouldn't be the same style. Can you use the generic black bullet?

**Comment by caldav** [comment]
The iframe section should be as follow:

```
<section class="p-strip is-bordered">
  <div class="row">
    <div class="col-12">
      <h2 id="register-section">Watch the webinar</h2>
    </div>
    <div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;"><script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : 6793, "language": "en-US", "commId" : 355328, "displayMode" : "standalone", "height" : "auto" }</script><script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script></div>
  </div>
</section>
```

File: templates/engage/openstack-security-webinar.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          Understand common practices to operate secure private clouds based on Openstack
+        </li>
+        <li class="p-list__item is-ticked">
+          Understand operational concerns associated with these practices 
+        </li>
+        <li class="p-list__item is-ticked">
+          Examine how automation can help address these concerns
+        </li>
+        <li class="p-list__item is-ticked">
+          Explore how Charmes Openstack can help deploy key security features:
```
Comment (caldav) [inline templates/engage/openstack-security-webinar.html]:
```suggestion
          Explore how Charmed Openstack can help deploy key security features:
```

File: templates/engage/openstack-security-webinar.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <li>
+              Offer encryption at rest as a service to private cloud users
+            </li>
+            <li>
+              Implement certificate lifecycle management to keep data secure in all scenarios
+            </li>
+          </ul>
+        </li>
+      </ul>
+      <p><a href="https://ubuntu.com/engage/openstack-security-webinar?utm_source=Takeover &utm_medium=Takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity" class="p-button--positive">Register for webinar</a></p>
```
Comment (caldav) [inline templates/engage/openstack-security-webinar.html]: This button needs to be taken out. The iframe webinar section (see my previous PR comments) should be added after the current one.

File: templates/engage/openstack-security-webinar.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+{% block title %}Private cloud security{% endblock %}
+
+{% block meta_description %}In regulated sectors, private cloud security needs to meet the highest requirements. This webinar explains how to deliver on these with an Openstack cloud.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1Qg3sgDrvH9w_sgJpb2fp5lyOA5R-xhGZM3aKcyuQT68/edit?ts=5cbee163#{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Private cloud security" subtitle="How Charmed OpenStack is decreasing common security concerns" class="p-strip--dark p-takeover--openstack"%}
```
Comment (caldav) [inline templates/engage/openstack-security-webinar.html]: Mobile header CTA

```suggestion
{% include "engage/shared/_header.html" with title="Private cloud security" subtitle="How Charmed OpenStack is decreasing common security concerns" class="p-strip--dark p-takeover--openstack" url="#register-section" cta="Watch the webinar" %}
```

**Pull request merged**